docs/Sprint-Review/sprint-38-summary.md

```diff
@@ -0,0 +1,62 @@
+# Sprint 38 Summary
+** 1/5/2021 - 1/18/2021 **
+
+## Sprint 38 Goals
```

Comment:
would it be possible to frame these as user stories going forward? @lfrohlich @valcollignon

Reply:
I don't think it will always fit but we can try!

Reply:
I'm not sure I understand, @ADPennington...